<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员页面</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUser">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>56</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUserMod">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbAdmin">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>提升为管理员</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbUser">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>设为普通用户</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbdelUser">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>删除用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>书本名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leBname">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWriter">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddBook">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>增加书目</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>书号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>书名</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lebookDel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnbookDel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnbookUpd">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>的留言</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMesUser">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>336</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMsgSea">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QTableView" name="tvMes">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>541</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMsgAll">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>显示全部留言</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>570</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>删除mid为</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDemid">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>560</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>的留言</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMesdel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>570</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>570</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>删除全部留言</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
